commcare_connect/opportunity/views.py

```diff
@@ -1632,3 +1653,118 @@ def user_visit_details(request, org_slug, opp_id, pk):
             is_program_manager=is_program_manager,
         ),
     )
+
+
+def opportunity_worker(request, org_slug=None, opp_id=None):
```

Comment:
nit: this might be cleaner as a class based view so we can break out the different parts (breadcrumbs, tabs, etc.)